perception/image_projection_based_fusion/docs/roi-cluster-fusion.md

```diff
@@ -32,18 +32,19 @@ The clusters are projected onto image planes, and then if the ROIs of clusters a
 
 ### Core Parameters
 
-| Name                        | Type  | Description                                                                      |
-| --------------------------- | ----- | -------------------------------------------------------------------------------- |
-| `use_iou_x`                 | bool  | calculate IoU only along x-axis                                                  |
-| `use_iou_y`                 | bool  | calculate IoU only along y-axis                                                  |
-| `use_iou`                   | bool  | calculate IoU both along x-axis and y-axis                                       |
-| `use_cluster_semantic_type` | bool  | if `false`, the labels of clusters are overwritten by `UNKNOWN` before fusion    |
-| `only_allow_inside_cluster` | bool  | if `true`, the only clusters contained inside RoIs by a detector                 |
-| `roi_scale_factor`          | float | the scale factor for offset of detector RoIs if `only_allow_inside_cluster=true` |
-| `iou_threshold`             | float | the IoU threshold to overwrite a label of clusters with a label of roi           |
-| `unknown_iou_threshold`     | float | the IoU threshold to fuse cluster with unknown label of roi                      |
-| `rois_number`               | int   | the number of input rois                                                         |
-| `debug_mode`                | bool  | If `true`, subscribe and publish images for visualization.                       |
+| Name                           | Type   | Description                                                                                                                                                                                    |
+| ------------------------------ | ------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `iou_mode`                     | string | select mode from 3 options {`iou`, `iou_x`, `iou_y`} to calculate IoU. <br> &emsp;`iou`: IoU along x-axis and y-axis <br> &emsp;`iou_x`: IoU along x-axis <br> &emsp;`iou_y`: IoU along y-axis |
+| `trust_distance`               | double | If the detected object's distance to frame_id is less than the threshold, the fusion will be processed                                                                                         |
+| `iou_x_use_distance_threshold` | double | if the detected object's distance to frame_id is is greater than the threshold, `iou_x` is used as default instead of `iou_mode`                                                               |
```

Comment:
The following is better to be prepared.
- iou_mode -> trust object iou mode
- trust_distance -> trust object range(distance)
- non trust object iou mode

Reply:
@yukkysaito I'm sorry for missed your comment.
Currently the `trust_distance` is used to limit range of `roi_cluster_fusion`.It mean there no fusion in longer than trust_distance. 

IMO, 
change: `iou_mode` -> `trust_object_iou_mode`
change: `trust_distance` -> `fusion_distance` 
add: `non_trust_object_iou_mode` (defult: iou_x)
change: `iou_x_use_distance_threshold` -> `trust_iou_mode_distance` (the distance to shift from trust to non-trush iou_mode"

Reply:
@yukkysaito As our discussion, the parameter will  look like this.

![image](https://github.com/autowarefoundation/autoware.universe/assets/94814556/ca67615d-9d7b-46cc-af8e-7262bfb5a8e6)

Reply:
LGTM :+1:

Reply:
@yukkysaito I reflect parameter renaming at https://github.com/autowarefoundation/autoware.universe/pull/5148/commits/21e40dfb37204ca27dbc56a1e19a99d12e4e4903